mlrun/feature_store/feature_vector.py

```diff
@@ -505,6 +505,13 @@ def get(self, entity_rows: List[Union[dict, list]], as_list=False):
                         and column != self.vector.status.label_column
                     ):
                         data[column] = None
+                aliases = self.vector.get_feature_aliases()
+                for column, alias in aliases.items():
+                    if (
+                        alias not in actual_columns
+                        and column != self.vector.status.label_column
+                    ):
+                        data[alias] = None
```

Comment:
We discussed removing this as it doesn't actually fix a test failure or another known issue.